project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=0.13.13
```

Comment:
Re: ` sbt.version`, gotcha, added in https://github.com/astorije/beer/commit/4bfd1d2c6e32bc20812fa475b2636942ddf194bf.

Re: `.idea`, I prefer to keep OS-specific and IDE-specific files out of my projects and use a global ignore list instead. For example, see [this](https://github.com/astorije/dotfiles/blob/81d66146b2d6194dced857f7ffce0b1b4d344239/git/gitignore_global) and [that](https://github.com/astorije/dotfiles/blob/81d66146b2d6194dced857f7ffce0b1b4d344239/git/gitconfig#L55). That's a personal preference but I prefer that over having to duplicate/maintain these in all project gitignore files :)